.gitignore

```diff
@@ -182,7 +182,7 @@ generated_imgs/
 
 # Custom model related artefacts
 variants.json
-models/
+/models/
```

Comment:
did you need this change to ignore changes in models/ ? I thought the existing one should have worked.

Reply:
It was ignoring the `models` folder fine, but it also seemed to be ignoring anything with `models` as a folder in the path, and git would keep complaining about it when I tried to add my changes for 'apps/stable_diffusion/src/**models**/model_wrappers.py'

I might just be my git installation or my editor being weird.

Reply:
I also noticed this with language models, didn't think much of it but this is probably the reason why that was happening. Thanks